alphagov/paas-cf
# [#114193485] Rename skip-final-snapshot Terraform variable

## What

… to `cf_db_skip_final_snapshot`, so that it is snake-case (instead of
kebab-case) and prefixed with `cf_db_` (so that the context is clearer).

This matches the neighbouring variables `cf_db_multi_az` and
`cf_db_backup_retention_period`.

This stood out to me and I thought it was worth refactoring.

## How to review

1. Configure your pipelines to use this branch:

        BRANCH=114193485-cf_db_skip_final_snapshot make dev
1. Run the `deploy-cloudfoundry` pipeline and confirm that there are no errors or changes to the RDS resources. 

## Who can review

Anyone but @dcarley
